src/endpoints/cart.js

```diff
@@ -67,6 +67,14 @@ class CartEndpoint extends BaseExtend {
     )
   }
 
+  CreateCart(cartData, token) {
```

Comment:
@petro97 just confirming, was there an option to create a cart with a specified cartID, or are ID's all controlled via the API?

Reply:
ID is an optional parameter https://elasticpath.atlassian.net/browse/MT-1022?focusedCommentId=228780